dotbot/controller.py

```diff
@@ -208,24 +208,20 @@ def on_command_move_raw(self, topic, payload):
             logger.warning("DotBot not found")
             return
         logger.info("Sending command")
-        header = ProtocolHeader(
+        header = Header(
+            version=PROTOCOL_VERSION,
+            type_=PacketType.DATA.value,
             destination=int(address, 16),
             source=int(self.settings.gw_address, 16),
```

Comment:
The real gateway address (`source`) will not be known by the top-level controller. I think it should not be set here (use a `None` as the default?).

Reply:
You are right but this is anticipating a bit. I'll think of a better solution when I'll start working on #148